tests/e2e/specs/modules/analytics/admin-bar.test.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * WordPress dependencies
+ */
+import { activatePlugin, createURL } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	setupAnalytics,
+	setSiteVerification,
+	setSearchConsoleProperty,
+	useRequestInterception,
+} from '../../../utils';
+import * as adminBarMockResponses from './fixtures/admin-bar';
+
+let mockBatchResponse;
+
+describe( 'Site Kit admin bar component display', () => {
+	beforeAll( async () => {
+		await activatePlugin( 'e2e-tests-auth-plugin' );
+		await activatePlugin( 'e2e-tests-admin-bar-visibility' );
+		await setSiteVerification();
+		await setSearchConsoleProperty();
+
+		await page.setRequestInterception( true );
+		useRequestInterception( ( request ) => {
+			if ( request.url().match( 'google-site-kit/v1/data/' ) ) {
+				request.respond( {
+					status: 200,
+					body: JSON.stringify( mockBatchResponse ),
+				} );
+			} else {
+				request.continue();
+			}
+		} );
+	} );
+
+	beforeEach( async () => {
+		mockBatchResponse = [];
+
+		await page.goto( createURL( '/hello-world' ), { waitUntil: 'load' } );
+	} );
+
+	it( 'loads Analytics data when the module is active', async () => {
```

Comment:
Does this also need a test for when analytics is setup and  the user navigates to Gutenberg?

Reply:
We could, but I don't think so. The admin bar app is the same everywhere, but it loads contextual data based on the post it's on so if it loads Analytics on the front, it will also load it in wp-admin. The AC only outlines these 4 scenarios but I'll defer to @felixarntz.